worldedit-core/src/main/java/com/sk89q/util/StringUtil.java

```diff
@@ -331,4 +332,37 @@ public static int getLevenshteinDistance(String s, String t) {
 
         return parsableBlocks;
     }
+
+    /**
+     * Splits a string respecting enclosing quotes.
+     *
+     * @param input the input to split.
+     * @param delimiter the delimiter to split on.
+     * @param open the opening quote character.
+     * @param close the closing quote character.
+     * @return a list of split strings.
+     */
+    public static List<String> split(String input, char delimiter, char open, char close) {

```

Comment:
This one is most likely to break other things. WE uses the method above this one to parse strings but it doesn't work with nested [] brackets. That broke using simplex in simplex